bazel/WORKSPACE

```diff
@@ -88,21 +88,22 @@ http_archive(
 http_archive(
         name = "mkl_dnn",
         build_file = "//:bazel/mkl_dnn.BUILD",
-        sha256 = "ed56652dd237deb86ee9bf102c18de5f2625c059e5ab1d7512c8dc01e316b694",
-        strip_prefix = "mkl-dnn-0.21.2",
+        sha256 = "0e9bcbc86cc215a84a5455a395ce540c68e255eaa586e37222fff622b9b17df7",
+        strip_prefix = "mkl-dnn-1.1.3",
         urls = [
-            "http://mirror.tensorflow.org/github.com/intel/mkl-dnn/archive/v0.21.2.tar.gz",
-            "https://github.com/intel/mkl-dnn/archive/v0.21.2.tar.gz",
+            # TODO: No official v1.x release have been posted to http://mirror.tensorflow.org yet
```

Comment:
@ashahba Is ngraph-core using v1.1.3? Last I checked they were using 1.0.4. Please check once.

Reply:
@sharathns93 They seem be using `v1.1.1`: https://github.com/NervanaSystems/ngraph/blob/master/cmake/external_mkldnn_v1.cmake#L26